diff --git a/external/Depth-Anything-3/pyproject.toml b/external/Depth-Anything-3/pyproject.toml
index 6a11c7a..c3b25ed 100644
--- a/external/Depth-Anything-3/pyproject.toml
+++ b/external/Depth-Anything-3/pyproject.toml
@@ -19,7 +19,7 @@ dependencies = [
     "einops",
     "huggingface_hub",
     "imageio",
-    "numpy<2",
+    "numpy",
      "opencv-python",
     "xformers",
     "open3d",
diff --git a/external/Depth-Anything-3/requirements.txt b/external/Depth-Anything-3/requirements.txt
index 878db3f..c3e6c75 100644
--- a/external/Depth-Anything-3/requirements.txt
+++ b/external/Depth-Anything-3/requirements.txt
@@ -5,7 +5,7 @@ torchvision
 einops
 huggingface_hub
 imageio
-numpy<2
+numpy
  opencv-python
 xformers
 open3d
